### DISCUSSION TITLE
Mocking - Cannot access '__wdio_import0' before initialization

### DISCUSSION THREAD

**Discussion by Medvedscak** [webdriverio/webdriverio#12320] [Q&A]
Hey, I'm trying to mock the useRef() of the 'react' for my tests like this (the similar way works with jest and I'm trying to achieve the same with WebdriverIO):

```
import { fn, mock } from '@wdio/browser-runner'

mock('react', async (originalModule) => {
  return {
    __esModule: true,
    ...originalModule,
    useRef: fn(),
  }
})

```
And I'm getting this import message even when I'm not trying to render anything (empty test case). I'm trying to understand what't the issue but the error doesn't point at some specific line (it says it's the closing line of the mock in this module). I can't find anything useful about this error on Google. Please advise

**Comment by christian-bromann** [comment]
Hey @Medvedscak , thanks for raising the issue!

Mind providing a minimal reproducible example, I would be happy to look into this.
